### DISCUSSION TITLE
[M-Z] (17 Giugno 2019) Esercizio 6

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#486] [Esami & Esoneri M-Z]
### Traccia

<img width="926" alt="image" src="https://github.com/user-attachments/assets/0f5f9488-44b2-4392-bc3e-06059241cec4">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190617.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190617.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/b1b2fa29-8e53-42f8-95b4-fa86f1e16ab2)
